kt-review.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+import random
+import sys
+
+def main():
+    """Conduct a knight's tour using a brute force method, representing the
+       board as a dict.
+    """
+
+    if not valid_input():
+        sys.exit("usage: knightstour4.py rows columns attempts")
+
+    rows = int(sys.argv[1])
+    cols = int(sys.argv[2])
+    attempts = int(sys.argv[3])
+
+    for attempt in range(attempts):
+        board = create_board(rows, cols)
+        knight_pos = [0, 0]
```

Comment:
i thought it would be better to have it as a list since the knight position needs to change so it needs to be mutable?